### PR TITLE
Added a HTTP POST example to the api-routes example

### PR DESCRIPTION
**Pull request by ankri**
For some reason it took me some time to figure out how to `HTTP POST` data to the newly introduced API Routes.

Maybe because I was so fixated on express-like syntax `app.get`, `app.post`, `app.put` that I thought you could only `GET` stuff with the the API routes.

To save others from falling down that rabbit hole I added a short snippet to the `api-routes` example. I also added a check and comment to reiterate that next automatically parses the body if the `content-type` is `application-json`

### CODE REVIEW

File: examples/api-routes/pages/api/people/index.js [hunk 1]
```diff
@@ -1,5 +1,14 @@
 import { people } from '../../../data'
 
 export default (req, res) => {
-  res.status(200).json(people)
+  if (req.method === 'POST') {
+    if (req.headers['content-type'].startsWith('application/json')) {
+      // req.body will be automatically parsed to JSON when the request's Content-Type is `application/json`
+      res.json(req.body)
```
Comment (timneutkens) [inline examples/api-routes/pages/api/people/index.js]: It's generally considered a bad practice to mirror input back into the response as it causes vulnerabilities, so I'd rather not display that in the example. Instead you can maybe return something else 👍

Comment (ankri) [reply examples/api-routes/pages/api/people/index.js]: Thank you for your suggestion. That totally makes sense. 

I messed up by `force-pushing` somewhere. I hope I fixed it and that the returned texts now make more sense than returning the `req.body`

File: examples/api-routes/pages/api/people/index.js [hunk 2]
```diff
@@ -1,5 +1,18 @@
 import { people } from '../../../data'
 
 export default (req, res) => {
-  res.status(200).json(people)
+  if (req.method === 'POST') {
+    if (req.headers['content-type'].startsWith('application/json')) {
+      // req.body will be automatically parsed to JSON when the request's Content-Type is `application/json`
+      res.send('The data you sent has been interpreted as application/json.')
+    } else {
```
Comment (timneutkens) [inline examples/api-routes/pages/api/people/index.js]: I don't think this else is needed.

**Comment by timneutkens** [comment]
I'm going to close this PR to prevent stacking up stale PRs, but we'd be happy to merge the update once the feedback is addressed 💯

**Pull request closed without merging**